src/test/scala/chisel3/internal/NameCollisionSpec.scala

```diff
@@ -64,15 +64,14 @@ class NameCollisionSpec extends ChiselFlatSpec with Utils {
   }
 
   it should "error on nameless ports being assigned default names" in {
-
-    (the[ChiselException] thrownBy extractCause[ChiselException] {
+    ((the[ChiselException] thrownBy extractCause[ChiselException] {
       ChiselStage.emitCHIRRTL(
         new Module {
           // Write to an output port that isn't assigned to a val, and so doesn't get prefixed
           IO(Output(UInt(8.W))) := 123.U
         },
         Array("--throw-on-first-error")
       )
-    }).getMessage should include("try making it a public field of the Module")
+    }).getMessage should include).regex("Assign .+ to a val, or explicitly call suggestName to seed a unique name")
```

Comment:
This ended up being the only change I could see related to the two-phase HasId naming in RawModule. Now that we are doing a pass of naming before `checkPorts`, the error message from check ports doesn't come up in this case. We do still get a nice error that basically says the same thing, so I think this is fine.